### PR TITLE
Feature/merge master into federation

### PR DESCRIPTION
**Pull request by ioangut**
## Checklist

* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [x] Make sure `mvn install` runs for the whole project and, if you touched any code in the respective service, submission and distribution service can be run with `spring-boot:run`

## Description

Merge master into federation branch.

### CODE REVIEW

File: services/submission/pom.xml [hunk 1]
```diff
@@ -67,6 +67,12 @@
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java-util</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java-util</artifactId>
+      <version>3.12.4</version>
+      <scope>test</scope>
+    </dependency>
```
Comment (hilmarf) [inline services/submission/pom.xml:75]: not required - please remove

File: services/submission/src/test/java/app/coronawarn/server/services/submission/controller/SubmissionControllerTest.java [hunk 2]
```diff
@@ -20,19 +20,8 @@
 
 package app.coronawarn.server.services.submission.controller;
 
-import static app.coronawarn.server.services.submission.controller.SubmissionPayloadMockData.VALID_KEY_DATA_2;
-import static app.coronawarn.server.services.submission.controller.SubmissionPayloadMockData.buildMultipleKeys;
-import static app.coronawarn.server.services.submission.controller.SubmissionPayloadMockData.buildMultipleKeysWithoutDSOS;
-import static app.coronawarn.server.services.submission.controller.SubmissionPayloadMockData.buildMultipleKeysWithoutDSOSAndTRL;
-import static app.coronawarn.server.services.submission.controller.SubmissionPayloadMockData.buildMultipleKeysWithoutTRL;
-import static app.coronawarn.server.services.submission.controller.SubmissionPayloadMockData.buildPayload;
+import static app.coronawarn.server.services.submission.controller.SubmissionPayloadMockData.*;
```
Comment (hilmarf) [inline services/submission/src/test/java/app/coronawarn/server/services/submission/controller/SubmissionControllerTest.java:23]: is this a checkstyle rule?
normally ...* imports should be avoided

**Review by hilmarf — CHANGES_REQUESTED** [review]

File: services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/diagnosiskeys/ProdDiagnosisKeyBundler.java [hunk 3]
```diff
@@ -20,6 +20,7 @@
 
 package app.coronawarn.server.services.distribution.assembly.diagnosiskeys;
 
+import static java.time.ZoneOffset.UTC;
```
Comment (hilmarf) [inline services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/diagnosiskeys/ProdDiagnosisKeyBundler.java:23]: unsused import

File: services/submission/src/test/java/app/coronawarn/server/services/submission/controller/RequestExecutor.java [hunk 4]
```diff
@@ -78,4 +85,32 @@ private HttpHeaders buildDefaultHeader() {
         .withoutCwaFake()
         .build();
   }
+
+  public static TemporaryExposureKey buildTemporaryExposureKey(
```
Comment (EugenM-SAP) [inline services/submission/src/test/java/app/coronawarn/server/services/submission/controller/RequestExecutor.java:89]: I believe these helper methods are not needed in RequestExecutor. They have been moved to SubmissionPayloadMockData in the federation branch. I think they should not be  "brought" in from master

**Review by EugenM-SAP — APPROVED** [review]

**Review by sorin-stefan-iovita-sap — APPROVED** [review]

**Review by fredrb — APPROVED** [review]

**Pull request merged**